### ISSUE TITLE
Could not find me.relex:photodraweeview:1.1.0.

### ISSUE THREAD

**Issue by luutruong** [merryjs/photo-viewer#133]
```
Searched in the following locations:
       - https://repo.maven.apache.org/maven2/me/relex/photodraweeview/1.1.0/photodraweeview-1.1.0.pom
       - file:/Users/xx/.m2/repository/me/relex/photodraweeview/1.1.0/photodraweeview-1.1.0.pom
       - file:/Users/xx/Documents/XenApp/node_modules/react-native/android/me/relex/photodraweeview/1.1.0/photodraweeview-1.1.0.pom
       - file:/Users/xx/Documents/xx/node_modules/jsc-android/dist/me/relex/photodraweeview/1.1.0/photodraweeview-1.1.0.pom
       - https://maven.google.com/me/relex/photodraweeview/1.1.0/photodraweeview-1.1.0.pom
       - https://dl.google.com/dl/android/maven2/me/relex/photodraweeview/1.1.0/photodraweeview-1.1.0.pom
       - https://www.jitpack.io/me/relex/photodraweeview/1.1.0/photodraweeview-1.1.0.pom
     Required by:
         project :app > project :merryjs_photo-viewer
```

Dependencies:
```
"@merryjs/photo-viewer": "^3.2.1"
"react": "17.0.2",
 "react-native": "0.65.1",
```

I got that error while building Android app.

**Comment by luutruong** [comment]
https://repo.maven.apache.org/maven2/me/relex/

There are no `photodraweeview` in the tree.

**Comment by luutruong** [comment]
Related: https://github.com/ongakuer/PhotoDraweeView/issues/82

**Comment by bang88** [comment]
@luutruong Please try 3.2.2 and let me known does it fixes your issues.

**Comment by luutruong** [comment]
@bang88 Did you commit here? Also lib you are using in iOS very old. Could you take your time to upgrade as well?

**Comment by bang88** [comment]
@luutruong Sorry  I just published a new version to npm `3.2.3` .

**Comment by bang88** [comment]
> @bang88 Did you commit here? Also lib you are using in iOS very old. Could you take your time to upgrade as well?

Will do it after `10.2` if time free
